libs/components/core/src/lib/modules/log/types/log-deprecation-args.ts

```diff
@@ -0,0 +1,28 @@
+import { SkyLogLevel } from './log-level';
+
+/**
+ * @internal
+ * Arguments used when logging that a class, property, function, etc. is depcrecated.
+ */
+export interface SkyLogDeprecationArgs {
```

Comment:
The name of the method is `deprecated`; we should probably name this interface (and the file name) to fit that naming convention. `SkyLogDeprecatedArgs`

Reply:
Done